mne/decoding/tests/test_time_gen.py

```diff
@@ -241,6 +241,10 @@ def test_generalization_across_time():
 
     # TODO JRK: test GAT with non-exhaustive CV (eg. train on 80%, test on 10%)
 
+    # Make CV with some empty tests to ensure that the folds are passed
+    gat.cv_.test_folds[gat.cv_.test_folds == 1] = 0
+    gat.predict(epochs)
```

Comment:
Ok I'll add a warning for empty test fold (which isn't recommended but shouldn't create an error) and raise an error for empty train fold (in which case we can't fit the estimator)